assets/wire/utils.ts

```diff
@@ -289,6 +289,9 @@ function getTextFromNode(node: Node | null): string {
     if (node.nodeType === Node.TEXT_NODE) {
         const textContent = node.textContent || '';
         return textContent.trim();
+    } else if (node.nodeType === Node.ELEMENT_NODE) {
+        const element = node as HTMLElement;
+        return element.innerText.trim();
```

Comment:
not sure if we need that trim, does it add some extra spaces?

Reply:
No, it doesn't add extra spaces; I'm just using this as a precautionary measure.